src/zoid/card-fields/prerender.jsx

```diff
@@ -8,7 +8,7 @@ type PrerenderedCardProps = {|
     height : ?number
 |};
 
-const DEFAULT_HEIGHT = 30;
+const DEFAULT_HEIGHT = 78;
```

Comment:
Does this impact the single card field height? Wondering if we need to give that one a different prerender height since it's only one text box.

Reply:
Yes, this is used by the single card field as well. From the [Figma designs](https://www.figma.com/file/78svOAGmEt7fk4OIU6Qde1/UP_-UDCC-Drop-in?node-id=472%3A3161), the single card field should have the same height as each multi card field. This is the height for each individual card field, with some extra to allow the focus drop shadow to display.